build.gradle

```diff
@@ -16,7 +16,6 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-import at.bxm.gradleplugins.svntools.tasks.SvnCheckout
 import org.apache.tools.ant.filters.ReplaceTokens
 import org.asciidoctor.gradle.AsciidoctorTask
 
```

Comment:
SvnCheckout and svntools should not be removed. They are still used by pullPluginSource and pullAllPluginsSource tasks which should also not be removed.